test/src/main/java/org/apache/accumulo/test/proxy/SimpleProxyBase.java

```diff
@@ -2077,7 +2077,8 @@ public void bulkImport() throws Exception {
 
     // Write an RFile
     String filename = dir + "/bulk/import/rfile.rf";
-    FileSKVWriter writer = FileOperations.getInstance().openWriter(filename, fs, fs.getConf(), null, DefaultConfiguration.getInstance());
+    FileSKVWriter writer = FileOperations.getInstance().openWriter().ofFile(filename, fs, fs.getConf())
+        .withTableConfiguration(DefaultConfiguration.getInstance()).execute();
```

Comment:
If default config were returned when withTableConfiguration were not called, then calls like this could be removed.